pkg/lib/cockpit-components-plot.jsx

```diff
@@ -431,7 +431,7 @@ export const SvgPlot = ({ title, config, plot_state, plot_id, className }) => {
                  // TODO: Figure out a way to handle a11y without entirely hiding the live-updating graphs
                  aria-hidden="true"
                  role="img"
-                 onMouseDown={plot_state.zoom_state && plot_state.zoom_state.enable_zoom_in ? start_dragging : null}
+                 onMouseDown={plot_state.zoom_state?.enable_zoom_in ? start_dragging : null}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18448-20230311-055523-7e3e6fe9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)